mscore/scorecmp/scorecmp.h

```diff
@@ -49,6 +49,8 @@ class ScoreComparisonTool : public QDockWidget {
       ScoreDiff* _diff = nullptr;
       RawScoreDiffModel* _rawModel = nullptr;
       ScoreDiffModel* _intelligentModel = nullptr;
+      
+      bool screenModeBefore = false;
```

Comment:
This variable name is a little ambiguous. How about `wasSplitScreen` instead?

Reply:
Yes, that's better. I'll change it.